README.md

```diff
@@ -4,6 +4,14 @@ Source code for the 2021 redesign of [cal-adapt.org](https://cal-adapt.org).
 
 This project was bootstrapped with the [Sapper framework](https://sapper.svelte.dev/) for [SvelteJS](https://svelte.dev).
 
+## Node version
```

Comment:
@melijimenez11 Not requesting any changes here, but in the future feel free to create a separate PR and merge changes like these directly into the `main`. Just a heads up is enough.